docs/02.API-REFERENCE.md

```diff
@@ -5303,30 +5303,28 @@ main (void)
 - [jerry_parse_and_save_function_snapshot](#jerry_parse_and_save_function_snapshot)
 
 
-## jerry_parse_and_save_literals
+## jerry_get_literals_from_snapshot
 
 **Summary**
 
-Collect the used literals from the given source code and save them into a specific file in a list or C format.
-These literals are generated by the parser, they are valid identifiers and none of them are magic string.
+Collect the used literals from the given snapshot and save them into a buffer in list or C format.
+None of these literals are magic strings. In C format only valid identifiers are collected.
```

Comment:
Is there a reason for the C limitation? It would not be bad to control this with a flag.

Reply:
(Can be done later if backward incompatible API changes can be avoided)

Reply:
The reason is that special unicode characters breaks the C format. It is deficiency of the current version (latest master) as well. I'd like to work on it in a followup if it is good for you.